requirements/framework.txt

```diff
@@ -3,10 +3,10 @@ aiofiles==23.2.1
 aiomysql==0.1.1
 httpx==0.27.0
 httpx-ntlm==1.4.0
-elasticsearch[async]==8.14.0
-elastic-transport==8.13.1
+elasticsearch[async]==8.15.0
+elastic-transport==8.15.0
 pyyaml==6.0.1
-cffi==1.16.0
+cffi==1.16.0=
```

Comment:
how......

Reply:
That was my attempt to fix the merge conflict, sorry 🤦

Reply:
I think I saw my name on it actually, that comment was to me haha